TestVectors/Ed25519Signature2020/sigBTC58EdSig.txt

```diff
@@ -0,0 +1 @@
+z2fa386qEULLhQvuusYwE1wfjb7i6p9Ba15m83CpZne9ZgsiZgX8bUV3Ucdx8atbUgyskBkeouMHA9ay2G8Yqeu6n
```

Comment:
If nothing else needs to change after the missing context is added, this is the expected value:

```suggestion
zveu47DychpPUMPjgyDdFynHQSrXWZLy75uPLn8TB6aSyFRRvJHW7BHL4Nkyg1sSQWQZTa5196X8jo7EZMWEjKuz
```